### PR TITLE
Remove prompts from commented examples

### PR DESCRIPTION
**Pull request by rhcarvalho**
Following the upstream style changes from Kubernetes.

Based on https://github.com/kubernetes/kubernetes/issues/13719 and https://github.com/kubernetes/kubernetes/pull/22177.

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@kargakis PTAL

**Comment by kargakis** [comment]
@smarterclayton I would like us to have this for 1.2 for parity with kubectl

**Comment by kargakis** [comment]
cc: @fabianofranz

**Comment by smarterclayton** [comment]
It's likely too late to make a change like this - it's not a must have, it's just a nice to have.

**Comment by fabianofranz** [comment]
You will have to `hack/update-generated-docs.sh` but apart that LGTM. 

There are no potentially breaking changes where so if https://github.com/kubernetes/kubernetes/pull/22177 landed before the version of kube we have then my vote is to have this to 1.2, but it's up to @smarterclayton to decide.

**Comment by rhcarvalho** [comment]
Thanks @fabianofranz! I've run `hack/update-generated-docs.sh` and updated my commit.

**Comment by smarterclayton** [comment]
Given size of change and lateness of schedule, we should only be merging
p0's or build/test fixes.

On Mon, Apr 11, 2016 at 10:27 AM, Rodolfo Carvalho <notifications@github.com

**Comment by rhcarvalho** [comment]
Rebased.

File: pkg/cmd/cli/cmd/wrappers.go [hunk 1]
```diff
@@ -310,29 +310,29 @@ foreground for an interactive container execution.  You may pass 'run/v1' to
...
   # Start a single instance of nginx, but overload the spec of the replication
   # controller with a partial set of values parsed from JSON.
-  $ %[1]s run nginx --image=nginx --overrides='{ "apiVersion": "v1", "spec": { ... } }'
+  %[1]s run nginx --image=nginx --overrides='{ "apiVersion": "v1", "spec": { ... } }'
 
   # Start a single instance of nginx and keep it in the foreground, don't restart it if it exits.
-  $ %[1]s run -i --tty nginx --image=nginx --restart=Never`
+  %[1]s run -i --tty nginx --image=nginx --restart=Never`
 
 	// TODO: uncomment these when arguments are delivered upstream
```
Comment (rhcarvalho) [inline pkg/cmd/cli/cmd/wrappers.go]: Does anybody know if this should be uncommented? The TODO was added in https://github.com/openshift/origin/pull/4299, last August.

Comment (kargakis) [inline pkg/cmd/cli/cmd/wrappers.go]: Yes, I think we can uncomment those now. Mind adding test cases for them, if we don't have already? (which most probably is the case:) )

**Comment by kargakis** [comment]
@rhcarvalho please rebase so we can move forward with this.

**Comment by rhcarvalho** [comment]
@kargakis rebased.

**Comment by kargakis** [comment]
LGTM [merge]

**Comment by rhcarvalho** [comment]
@kargakis I fixed the generated docs, was missing a re-run after I replaced some '$ ' in new code after the rebase.

**Comment by kargakis** [comment]
[test]

**Comment by kargakis** [comment]
https://github.com/openshift/origin/issues/8865 and redis image flakes

[merge]ing

**Pull request merged**